### DISCUSSION TITLE
Double click yellow triangle to just solve it

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9908] [Editor]
The yellow triangle, let users double click it and auto-do the solution, single click to show the popup like currently, hover for description like currently too.

EXAMPLE: Make Area2d node, yellow triangle appears, double click it and it just makes the collision shape child node automatically. Imagine how much better that would be, effortless, and the first thing every user tries anyway.

Thoughts?

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/2519
